src/components/Vuetify/mixins/icons.js

```diff
@@ -0,0 +1,110 @@
+// Maps internal Vuetify icon names to actual Material Design icon names.
+const ICONS_MATERIAL = {
```

Comment:
about the naming:
 - `rchecked` etc - imho no need to create such not-so-easy-to-guess abbreviations, i'd suggest `radionOn`, `radioOff`, `checkboxOn`, `checkboxOff`, `checkboxIndeterminate`
- `dot` - the name should say what it is, not how it looks like, so maybe `delimiter`?
- `append` - the name should say what it is, now where it is located - maybe use `expand`?
- `subgroup`  as a name of the `v-select` icon - maybe `dropdown`? i think `subgroup` for `v-list-group` is ok. So there would be 3 "arrow down" icons: `subgroup`, `dropdown` and `expand`

Reply:
I new when I added the missing `rchecked` etc that it was unlikely to survive that way. ;)

In general, when I first added each icon, I wasn't sure to what extent it was currently used. So I tried to preserve the existing naming, at least where it wasn't very iconfont-specific, or simplify/shorten it where it was.  

I'm definitely with you on most of the above and in general. Now that I've produced the list above, the intent is a lot clearer and it's a good chance to review all that naming.

I think that if we're okay with two-word camel-case names, something like contextAction (e.g. radioOn) is great.  And On/Off is nice and short and clear, better than Checked.  So those first 5 I'll do today without needing any further comment from anyone else, I think that would be widely accepted.

The `dot` case isn't really how it looks, and it's not a delimiter.  It's the placeholder blob on the bottom of the [carousel](https://vuetifyjs.com/en/components/carousels) representing an image or item in the list of carousel items. But since it was just a placeholder, circular and meant to be filled in, I just used `dot` initially.  Maybe `carouselItem` or just `placeholder` or `blob` or `placeholderBlob` if it gets used for something else (not currently).  Something like `carouselPlaceholder` is kinda long for my taste.  Yours too, or okay?  I'm kinda leaning towards `placeholder` here now, although I don't really dislike `dot` or `blob` for his one. 😀 Comments?

Yes, the `append` icon (in VListGroups) is an _expand_, and appears to perform the same operation as the icon in expansion panels, and should probably removed entirely and replaced with the existing `expand` icon (tied together). 

Completely agreed on `subgroup`, I think the suggestion is to split the two use cases into `subgroup` and `dropdown`, and I love that idea and I can do that today as well.

Reply:
About the `delimiter` - i don't like the name too, but the prop name is `delimiterIcon` so i think the icon name should be in line with the prop name (somehow I feel that this idea is valid with this particular icon, but not with the `append`...)

Reply:
In that case I'm also totally in favor of `delimiter`, and that solves that one nicely. Thanks for the update/info on that one.  I'll have the commit with all the renames done soon for further review.